test/basic/basic.go

```diff
@@ -93,6 +93,22 @@ func (s *TestSuite) TestBasics(t *testing.T) {
 	resp, err = subClient.Recv() // Now get the addition notification
 	assert.NoError(t, err)
 	assert.Equal(t, "201", resp.GetUpdate().Update[0].Path.Elem[1].Key["port"])
+
+	wg2 := sync.WaitGroup{}
+	wg2.Add(5) // setting counter to 5 (until at least 5 hosts are found..)
+
+	for i, device := range dresp.Devices {
+		go func(id int, d *simapi.Device) {
```

Comment:
This is not needed since the pipeline config is setup earlier by the same test.

Reply:
removed